runtime/Makefile

```diff
@@ -353,7 +353,10 @@ $(foreach object_type, $(object_types), \
 %_libasmrunpic.o: %.S
 	$(ASPP) $(ASPPFLAGS) $(SHAREDLIB_CFLAGS) -o $@ $<
 
-%.obj: %.asm
+domain_state.inc: caml/domain_state.tbl gen_domain_state_inc.awk
```

Comment:
`runtime/domain_state.inc` needs to go in `.gitignore`

Reply:
Done.